tsconfig.json

```diff
@@ -1,24 +1,20 @@
 {
-  "compilerOptions": {
-    "declaration": true,
-    "target": "es2020",
-    "module": "commonjs",
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "sourceMap": true,
-    "noImplicitAny": false,
-    "removeComments": true,
-    "preserveConstEnums": true,
-    "noEmitHelpers": true,
-    "outDir": "dist",
```

Comment:
Changing the `outDir=dist` in favor of `dist/mjs` and `dist/cjs` is a good improvement, but also a breaking change. 
We can publishing it as 3.0.0 to warn consumers and preserve SemVer semantics.

Reply:
If you can change the version number, please. I think we are all done and ready to merge.

Reply:
Sounds good!

A nice script around this is `semantic-release` - you can see an example here https://github.com/jonluca/har-to-openapi/blob/main/.github/workflows/release.yml

It's nice because you never have to update package.json yourself, and it deploys automatically based on your commits. Not saying we need to include it here, but something nice to keep in mind.

Updated the diff with 3.0.0

Reply:
Good point, thanks!